src/frontend/apps/desk/src/pages/teams/[id].tsx

```diff
@@ -26,21 +27,16 @@ interface TeamProps {
 }
 
 const Team = ({ id }: TeamProps) => {
-  const { t } = useTranslation();
-  const { data: team, isLoading, isError } = useTeam({ id });
+  const { data: team, isLoading, isError, error } = useTeam({ id });
+  const navigate = useNavigate();
 
-  if (isError) {
-    return (
-      <Text
-        $align="center"
-        $justify="center"
-        $height="100%"
-        $theme="danger"
-        $textAlign="center"
-      >
-        {t('Something bad happens, please retry.')}
-      </Text>
-    );
+  if (isError && error) {
```

Comment:
Why not something as follows? 

```suggestion
  if (isError) {
```

Reply:
[265e0b4](https://github.com/numerique-gouv/people/pull/52/commits/265e0b46278238b9022f9be0f949f7e4ae13ebf7)